src/legacy-theme/js/colors/dark.ts

```diff
@@ -0,0 +1,602 @@
+const colors = {
```

Comment:
the most pedantic nitpick, sorry!: the path should be `legacy-theme/ts/`

```diff
- src/legacy-theme/js/colors/dark.ts
+ src/legacy-theme/ts/colors/dark.ts
                    ↑ 

```

Reply:
Fixed ✅